modules/core/src/main/java/io/fluo/core/util/Halt.java

```diff
@@ -15,10 +15,14 @@
  */
 package io.fluo.core.util;
 
-import org.apache.log4j.Logger;
+import org.slf4j.Logger;
```

Comment:
Its kinda neat that by doing this analysis you found this lurking `log4j` dependency.